### PR TITLE
Projections route and Spinner

### LINKED ISSUE #825: Create new projections route and populate with placeholder spinner
**What needs to be done? Why does it need to be done?**
We need to add a new route for the projection views: `/facilities/projections/`

We also would like to standardize the UI for chart reloading. On a given chart that is loading, this should be the UI. [Mocks](https://www.figma.com/file/SOdY6JujMpbawgJ4jjDmE8/Population-Projection-MVP?node-id=866%3A47)

The goal of this task is to create this new route and populate it with a single loading component. Don't add any other components to the view. This loading indicator will spin forever as we aren't actually loading any data.

The loading component should be called `CoreLoadingIndicator.js`. The spinner should rotate once per second for now. Styles for the spinner can be found in the Figma files linked above.

### PR DESCRIPTION
**Pull request by MeloncholyShiro**
## Description of the change

I hope I understood the task correctly. I have implemented a loading circle with animation, and at the same time made a new path, and the item in the side menu Core Layout, not quite understand what I meant by the empty component, so just in case I did not add directly inside the component <PageTemplate>, I would be glad if you correct me :)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #825 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/components/CoreLoadingIndicator.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  width: $RING-SIZE;
+  height: $RING-SIZE;
+
+  &__circle {
+    box-sizing: border-box;
+    display: block;
+    position: absolute;
+    border: 6px solid #99b3b2;
+    border-radius: 50%;
+    animation: loading-ring 1s cubic-bezier(0.5, 0, 0.5, 1) infinite;
```
Comment (Catacola) [inline src/components/CoreLoadingIndicator.scss:34]: I like the choice of `cubic-bezier`! It's not what I was expecting, but upon seeing it, I think it's a good fit. Thanks!

File: src/components/CoreLoadingIndicator.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+
+  &__title {
+    padding-top: 15px;
+    text-align: center;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
```
Comment (Catacola) [inline src/components/CoreLoadingIndicator.scss:13]: Please convert the `px` to `rem` assuming a element size of 16px.

File: src/components/CoreLoadingIndicator.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    letter-spacing: -0.01em;
+  }
+}
+
+$RING-SIZE: 48px;
```
Comment (Catacola) [inline src/components/CoreLoadingIndicator.scss:18]: 3rem

**Review by Catacola — CHANGES_REQUESTED** [review]
Looking very nice!

**Comment by daschi** [comment]
@MeloncholyShiro Thank you for your work on this, to keep it moving forward with some other work I'm doing, I'm going to take over this branch and merge it with my other work.

**Review by Catacola — APPROVED** [review]
🔥 Ze 🚀

**Pull request merged**